draft-ietf-quic-transport.md

```diff
@@ -1721,7 +1721,12 @@ it can associate the peer's response with the corresponding PATH_CHALLENGE.
 ## Path Validation Responses
 
 On receiving a PATH_CHALLENGE frame, an endpoint MUST respond immediately by
-echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame.
+echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame,
+unless it has PATH_RESPONSE frames buffered for the same destination connection
+ID and wishes to limit memory consumtion, for example when congestion limited.
```

Comment:
This example is inconsistent with the advice of just using one congestion controller so we might want to skip it.

Reply:
I don't know that it's entirely inconsistent, but sure can remove it.

Reply:
Removed in d0d147a